.github/workflows/auto-merge.yml

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (c) 2019-2020, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# A workflow to keep BASE branch up-to-date from HEAD branch
+name: auto-merge HEAD to BASE
+
+on:
+  schedule:
+  - cron: '0 * * * *' # hourly
```

Comment:
is there any restrictions on how frequently we can run?

Reply:
there is (pricing)[https://github.com/pricing] for how many actions we can use monthly, Im not quite sure which plan we are using :)

Reply:
Rather than have this run every hour and burn up action-minutes with no benefit when nothing is merged during the hour, wouldn't it be better to trigger on a push to the branch?  That way we only run an action when we know there's work to do.

Reply:
@jlowe 
actually I was considering to trigger it daily, so it would pick up all merged commits in a single auto-merge PR.
`push` based trigger is also good, but one-new-PR per commit. Any thoughts on those two?

Reply:
make sense. I will adjust the config to push based. Thanks!